src/components/Demo.js

```diff
@@ -0,0 +1,33 @@
+import * as React from "react";
+import Title from "@components/typography/Title";
+
+import {
+  demoContainerPSectionTop,
+  demoContainerSectionTop,
+  demoSectionTop,
+} from "@styles/demo.module.css";
+
+
+const Demo = ({
+  children,
+  title,
+  subtitle,
+  ...props
+}) => {
+  const demo = props.demo || demoSectionTop;
+  const demoContainer = props.demoContainer || demoContainerSectionTop;
+  const demoContainerHeader = props.demoContainerHeader || undefined;
+  const demoContainerP = props.demoContainerP || demoContainerPSectionTop;
```

Comment:
Swoją drogą poprawiłbym lekko nazewnictwo. `demoContainerPSectionTop` nic konkretnego nie mówi :p Może dlatego, że nie wiem, na której stronie jestem, ale to jest temat jeszcze na przyszłe CR.